### PR TITLE
[CodeClean] missing lock

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Fix svace issues, add missing lock.

### CODE REVIEW

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 1]
```diff
@@ -3096,16 +3096,17 @@ nnstreamer_filter_shared_model_replace (void *instance, const char *key,
...
-    ml_loge ("The shared model representation is not supported properly!");
-    return;
-  }
   if (!key) {
     ml_loge ("The key should NOT be NULL!");
     return;
   }
 
   G_LOCK (shared_model_table);
+  if (!shared_model_table) {
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:3105]: Can you do G_LOCK if shared_model_table is NULL?

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_filter/tensor_filter_common.c:3105]: Internally it is defined using glib macro G_LOCK_DEFINE_STATIC (shared_model_table).

Comment (myungjoo) [reply gst/nnstreamer/tensor_filter/tensor_filter_common.c:3105]: Oh ok... G_LOCK(name) does not use a variable named ```name```, but ```g__name__lock```.

**Review by myungjoo — APPROVED** [review]

**Review by again4you — APPROVED** [review]
LGTM!

**Review by niley7464 — APPROVED** [review]
LGTM!

**Pull request merged**